playbooks/robusta_playbooks/node_disk_analysis.py

```diff
@@ -6,8 +6,13 @@
 import humanize
 
 
+class DiskAnalyzerParams(ActionParams):
+    show_pod_level_details: bool = True
+    show_container_level_details: bool = False
+
+
 @action
-def node_disk_analyzer(event: NodeEvent):
+def node_disk_analyzer(event: NodeEvent, params: DiskAnalyzerParams):
```

Comment:
Would update the docs about the configuration options, and what's chosen by default